test/functionalTest/cases/1048_subscription_cache/refresh_cache_wins_notif_success.test

```diff
@@ -219,9 +219,9 @@ Date: REGEX(.*)
             "http": {
                 "url": "http://localhost:9997/notify"
             },
-            "lastNotification": "202REGEX(.*)",
-            "lastSuccess": "202REGEX(.*)",
-            "lastSuccessCode": 200,
+            "lastNotification": "2050-01-01T10:00:00.000Z",
+            "lastSuccess": "2050-01-01T10:00:00.000Z",
+            "lastSuccessCode": 211,
```

Comment:
This shouldn't change.

Changes in this PR should be only in HTTP headers order as far as I understand.

I'm fact, this test is failing as seen at https://github.com/telefonicaid/fiware-orion/actions/runs/4919583948/jobs/8876245156

```
----------- Failing tests ------------------
  o  1048_subscription_cache/refresh_cache_wins_notif_disabled.test
  o  1048_subscription_cache/refresh_cache_wins_notif_success.test
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4321/commits/c7478f401c0466246950316809c9ec5521bb1ee1